File: oss/src/main/java/net/yan100/compose/oss/properties/MinioProperties.java
package net.yan100.compose.oss.properties;

import lombok.Data;

/**
 * minio 配置项
 *
 * @author TrueNine
 * @since 2023-02-21
 */
@Data
public class MinioProperties {
  private Boolean enable = false;
  private Boolean enableHttps = false;
  private String endpointHost = "localhost";
  private Integer endpointPort = 9000;
  private String accessKey;
  private String secretKey;
}

File: oss/src/main/kotlin/net/yan100/compose/oss/properties/MinioProperties.kt
package net.yan100.compose.oss.properties

/**
 * minio 配置项
 *
 * @author TrueNine
 * @since 2023-02-21
 */
class MinioProperties {
  var enable = false
  var enableHttps = false
  var endpointHost = "localhost"
  var endpointPort = 9000
  var accessKey: String? = null
  var secretKey: String? = null
}
